.github/workflows/gems-bump-version.yml

```diff
@@ -0,0 +1,71 @@
+name: Gems - Bump Version
+on:
+  workflow_dispatch:
+    inputs:
+      version_type:
+        description: "Version Type?"
+        required: true
+        type: choice
+        options:
+          - minor
+          - patch
+        default: "minor"
```

Comment:
When triggered via Cron, it'll run using `minor` with the assumption that new features got added since the most recent release. If that's not true, at that point we can always manually trigger a `patch` release instead. But I doubt we'll even pay attention to it much.